description.md

```diff
@@ -18,8 +17,34 @@ Each topic in this file contains a small description and links to the files cove
 
 ### Assignments
 
-### Drills
+Assignments are practical activities that learners undertake individually or in teams over the span of multiple weeks.
+The primary objective of an assignment is to create a compelling scenario that allows learners to apply the concepts they have learned.
+
+#### Support
+
+Assignments are typically meant for individual work, but they often require support from trainers.
```

Comment:
You should also cover team-based assignments. VMChecker accommodates them flawlessly, as it's based on GitLab.

Reply:
I can provide a general view of this as we have not experienced this idea.